src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -3162,6 +3162,26 @@
     ("SveCreateTrueMaskTest.template",    new Dictionary<string, string> { ["TestName"] = "Sve_CreateTrueMaskUInt32",                                                                              ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "CreateTrueMaskUInt32",                                                 ["RetVectorType"] = "Vector",    ["RetBaseType"] = "UInt32",                                   ["Op1Type"] = "SveMaskPattern"}),
     ("SveCreateTrueMaskTest.template",    new Dictionary<string, string> { ["TestName"] = "Sve_CreateTrueMaskUInt64",                                                                              ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "CreateTrueMaskUInt64",                                                 ["RetVectorType"] = "Vector",    ["RetBaseType"] = "UInt64",                                   ["Op1Type"] = "SveMaskPattern"}),
 
+    // ("SveGatherVectorVectorBases.template", new Dictionary<string, string> { ["TestName"] = "Sve_GatherVector_Bases_float_uint",                                                                   ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "GatherVector",                                                         ["RetVectorType"] = "Vector",    ["RetBaseType"] = "Single",  ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "Single", ["Op2VectorType"] = "Vector",    ["Op2BaseType"] = "UInt32",                                                              ["LargestVectorSize"] = "64", ["NextValueOp1"] = "Helpers.getMaskSingle()",      ["NextValueOp2"] = "TestLibrary.Generator.GetUInt32()",                    ["NextValueBase"] = "TestLibrary.Generator.GetSingle()",                       ["ValidateIterResult"] = "result[i] != (firstOp[i] == 0 ? 0 : *(Single*)(secondOp[i]))",}),
```

Comment:
> uncomment?

This is the issue here: https://github.com/dotnet/runtime/pull/103159#issuecomment-2154623775

It doesn't work because the addresses are 32bit.

Reply:
ah, didn't see that comment earlier.